client/src/components/Search/index.js

```diff
@@ -0,0 +1,74 @@
+import DateFnsUtils from '@date-io/date-fns';
+import { MuiPickersUtilsProvider, KeyboardDatePicker} from '@material-ui/pickers';
+import {Wrapper} from './styled'
+
+const Search = () => {
+
+    const [arrivalDate,setArrivalDate]=useState(new Date());
+    const handleDateChange=()=>{};
+
+
+    return (
+        <Wrapper>
+            <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                <Grid container>
+                    <Grid item xs={12} sm={2}>    
+                        <Grid className='title'>From</Grid>
+                        <Grid className='data'>Torono</Grid>
+                    </Grid>
+                    <Divider orientation="vertical" flexItem />
+                    <Grid item xs={12} sm={2}>
+                        <Grid className='title'>Where to go</Grid>
+                        <Grid className='data'>Bali</Grid>
+                    </Grid>
+                    <Divider orientation="vertical" flexItem />
+                    <Grid item xs={12} sm={2}>
+                        <KeyboardDatePicker
+                            disableToolbar
+                            variant="inline"
+                            format="MM/dd/yyyy"
+                            margin="normal"
+                            id="date-picker-inline"
+                            label="Arrival"
+                            value={arrivalDate}
+                            onChange={handleDateChange}
+                            KeyboardButtonProps={{
+                            'aria-label': 'change date',
+                        }}
+                        />
+                    </Grid>
+                    <Divider orientation="vertical" flexItem />
+                    <Grid item xs={12} sm={2}>
+                    <KeyboardDatePicker
+                            disableToolbar
+                            variant="inline"
+                            format="MM/dd/yyyy"
+                            margin="normal"
+                            id="date-picker-inline"
+                            label="Departure"
+                            value={arrivalDate}
+                            onChange={handleDateChange}
+                            KeyboardButtonProps={{
+                            'aria-label': 'change date',
+                        }}
+                        />
+                    </Grid>
+                    <Divider orientation="vertical" flexItem />
+                    <Grid item xs={12} sm={2}>
+                        <Grid className='title'>Travellers</Grid>
+                        <Grid className='data'>2 travellers</Grid>
+                    </Grid>
+                    <Divider orientation="vertical" flexItem />
+                    <Grid item xs={12} sm={2}>
+                        <Button variant='contained'> Search</Button>
+                    </Grid>   
+                </Grid>
+            </MuiPickersUtilsProvider>
+        </Wrapper>
+    )
+}
+
```

Comment:
I wonder if your sizing problem is because your dividers are not inside any Grid `item`s, and so the Grids all add up to 12, but there's the extra dividers and they throw it off. Try putting the dividers inside the Grid `item`s and see if that works. For example:
```
                    <Grid item xs={12} sm={2}>
                       <Divider orientation="vertical" flexItem />
                       <Button variant='contained'> Search</Button>
                    </Grid>   
```

Reply:
It works for the layout now, but in the meantime, the line of dividing disappears.